clippy_lints/src/use_self.rs

```diff
@@ -25,7 +26,12 @@ use if_chain::if_chain;
 /// name
 /// feels inconsistent.
 ///
-/// **Known problems:** None.
+/// **Known problems:**
+/// - Does not trigger within locally defined macros (#2098)
```

Comment:
I don't think we should list false negatives here.

Reply:
Ah, makes sense.